dev-tools/mage/build.go

```diff
@@ -70,6 +70,8 @@ func DefaultBuildArgs() BuildArgs {
 	} else {
 		// Strip all debug symbols from binary (does not affect Go stack traces).
 		args.LDFlags = append(args.LDFlags, "-s")
+		// Remove all file system paths from the compiled executable, to improve build reproducibility
+		args.ExtraFlags = append(args.ExtraFlags, "-trimpath")
```

Comment:
You could move it to the `args` declaration, that way it'd add the `-trimpath` to every build, not only the non-dev ones.

Reply:
Putting it here instead of making it a default was done on purpose.
E.g. if you replace a dependency in dev mode to a local checkout, printed stack traces and panics will hint you to the replacement if something doesn't work out. So this is to make sure a developer is aware from which source something is added.